include/seqan3/search/algorithm/search.hpp

```diff
@@ -55,10 +55,9 @@ template <FmIndex index_t, typename queries_t, typename configuration_t>
             (std::ranges::ForwardRange<queries_t> && std::ranges::RandomAccessRange<value_type_t<queries_t>>)) &&
         detail::is_type_specialisation_of_v<remove_cvref_t<configuration_t>, configuration>
 //!\endcond
-inline auto search(index_t const & index, queries_t && queries, configuration_t const & cfg)
+inline auto search(queries_t && queries, index_t const & index, configuration_t const & cfg)
```

Comment:
double check the documentation of this interface

Reply:
I meant that there is a TODO in it

Reply:
It's still there :expressionless:

Reply:
I'll do a separate docs PR